src/core/server/saved_objects/export/sort_objects.test.ts

```diff
@@ -199,71 +199,71 @@ Array [
       },
     ];
     expect(sortObjects(docs)).toMatchInlineSnapshot(`
-Array [
-  Object {
-    "attributes": Object {},
-    "id": "1",
-    "references": Array [],
-    "type": "index-pattern",
-  },
-  Object {
-    "attributes": Object {},
-    "id": "2",
-    "references": Array [
-      Object {
-        "id": "1",
-        "name": "ref1",
-        "type": "index-pattern",
-      },
-    ],
-    "type": "search",
-  },
-  Object {
-    "attributes": Object {},
-    "id": "3",
-    "references": Array [
-      Object {
-        "id": "2",
-        "name": "ref1",
-        "type": "search",
-      },
-    ],
-    "type": "visualization",
-  },
-  Object {
-    "attributes": Object {},
-    "id": "4",
-    "references": Array [
-      Object {
-        "id": "1",
-        "name": "ref1",
-        "type": "index-pattern",
-      },
-    ],
-    "type": "visualization",
-  },
-  Object {
-    "attributes": Object {},
-    "id": "5",
-    "references": Array [
-      Object {
-        "id": "3",
-        "name": "ref1",
-        "type": "visualization",
-      },
-      Object {
-        "id": "4",
-        "name": "ref2",
-        "type": "visualization",
-      },
-    ],
-    "type": "dashboard",
-  },
-]
-`);
+      Array [
+        Object {
+          "attributes": Object {},
+          "id": "1",
+          "references": Array [],
+          "type": "index-pattern",
+        },
+        Object {
+          "attributes": Object {},
+          "id": "2",
+          "references": Array [
+            Object {
+              "id": "1",
+              "name": "ref1",
+              "type": "index-pattern",
+            },
+          ],
+          "type": "search",
+        },
+        Object {
+          "attributes": Object {},
+          "id": "3",
+          "references": Array [
+            Object {
+              "id": "2",
+              "name": "ref1",
+              "type": "search",
+            },
+          ],
+          "type": "visualization",
+        },
+        Object {
+          "attributes": Object {},
+          "id": "4",
+          "references": Array [
+            Object {
+              "id": "1",
+              "name": "ref1",
+              "type": "index-pattern",
+            },
+          ],
+          "type": "visualization",
+        },
+        Object {
+          "attributes": Object {},
+          "id": "5",
+          "references": Array [
+            Object {
+              "id": "3",
+              "name": "ref1",
+              "type": "visualization",
+            },
+            Object {
+              "id": "4",
+              "name": "ref2",
+              "type": "visualization",
+            },
+          ],
+          "type": "dashboard",
+        },
+      ]
+    `);
   });
 
-  test('detects circular dependencies', () => {
+  test('should not fail on circular dependencies', () => {
     const docs = [
       {
         id: '1',
```

Comment:
Can we add a slightly more complicated test, such as
(object -> refs to)
1 -> 2, 3
2 -> 1, 3
3 -> 2, 4
4 -> 1

Reply:
done